tasks/skylab/FastqProcessing.wdl

```diff
@@ -88,10 +88,10 @@ task FastqProcessing {
     CODE)
 
     # use the right UMI length depending on the chemistry
-    if [ "~{chemistry}" == "tenX_v2" ]; then
+    if [ "~{chemistry}" == "2" ]; then
```

Comment:
@jway was there a cleaner way to do this since I took in the chemistry as an Int and in this code it's being used as a string in quotes?